cpp/oneapi/dal/backend/linalg/loops.hpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
```

Comment:
Where the content of that file planned to be used? Tests only or somewhere else too?

Reply:
All linalg stuff is in test/engine now